src/System.IO.Pipelines/ref/System.IO.Pipelines.csproj

```diff
@@ -16,4 +16,10 @@
   <ItemGroup>
     <ProjectReference Include="..\..\System.Buffers\ref\System.Buffers.csproj" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp'">
```

Comment:
@stephentoub isn't this in .NET Core >= 3.x?

Reply:
ManualResetValueTaskSourceCore? Yes.

Reply:
Does a corefx package targeting netcoreapp3.0+ still need to include a reference to `Microsoft.Bcl.AsyncInterfaces` even though its in the runtime?

Reply:
No